Source/ACE/Entity/Monster.cs

```diff
@@ -23,6 +25,74 @@ public Monster(AceObject aceO)
             else
                 this.WeenieClassid = (ushort)(aceO.WeenieClassId - 0x8000);
 
```

Comment:
TBH, I just noticed that there was an issue with monsters showing up on radar, so I c&p'd from DebugObject which corrected that. Didn't go beyond that. I don't get what the weenieclassid ushort stuff is either.